src/kinsky/client.clj

```diff
@@ -234,27 +235,33 @@
     (fn? x)      (x)
     :else        x))
 
-(defn ^Serializer ->serializer
+(defn ->serializer
+  ^Serializer
   [x]
   (cond
     (keyword? x) (if-let [f (serializers x)]
                    (f)
                    (throw (ex-info "unknown serializer alias" {})))
-    (fn? x)      (x)
+    (fn? x)     (x)
     :else        x))
 
-(defn ^Map opts->props
-  "Kakfa configs are now maps of strings to strings. Morph
-   an arbitrary clojure map into this representation."
-  [opts]
-  (into {} (for [[k v] opts] [(name k) (str v)])))
-
-(defn ^ConsumerRebalanceListener rebalance-listener
+(defn opts->props
+  "Kakfa configs are now maps of strings to strings. Morph an arbitrary
+  clojure map into this representation.  Make sure we don't pass
+  options that are meant for the driver with "
```

Comment:
with what? :)

Reply:
whoops :) It should be fixed now. thanks